src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -62,8 +65,16 @@ const fillAddressForm = (component: any, address: Address) => {
 describe("Payment", () => {
   let stripeMock
   let testProps: PaymentProps
+
+  const getWrapper = (props, breakpoint: Breakpoint = "xs") => {
+    return mount(
+      <MockBoot breakpoint={breakpoint}>
+        <PaymentRoute {...props} />
+      </MockBoot>
+    )
+  }
+
   beforeEach(() => {
-    console.error = jest.fn() // Silences component logging.
```

Comment:
@damassi This was masking actual errors, so removed it.

* At first a render error occurred that failed rendering due to trying to pass the `hidePostalCode` prop that was being passed to a mocked `CardElement` that was returning a `div` element.
* Then the actual issue showed up, which is `Warning: Can't call setState (or forceUpdate) on an unmounted component.`

Looking into that issue is up next.

Reply:
Ugh! just like we talked about the other day. I suspect there might be a few other examples of `console.log` / `console.error` being consumed around reaction, might be good to grep other tests.